tsconfig.json

```diff
@@ -2,10 +2,25 @@
   "compilerOptions": {
     "baseUrl": ".",
     "paths": {
-      "react-native-screens": ["./src"],
-      "react-native-screens/native-stack": ["./src/native-stack/index.tsx"],
+      "react-native-screens": ["src"],
+      "react-native-screens/native-stack": ["src/native-stack/index.tsx"],
       "react-native-screens/reanimated": ["src/reanimated/index.tsx"],
-      "react-native-screens/gesture-handler": ["src/gesture-handler/index.tsx"]
+      "react-native-screens/gesture-handler": ["src/gesture-handler/index.tsx"],
+      "@react-navigation/bottom-tabs": [
+        "link:../react-navigation/packages/bottom-tabs/src"
+      ],
+      "@react-navigation/core": ["link:../react-navigation/packages/core/"],
+      "@react-navigation/elements": [
+        "link:../react-navigation/packages/elements/src"
+      ],
+      "@react-navigation/native": ["link:../react-navigation/packages/native/"],
+      "@react-navigation/native-stack": [
+        "react-navigation/packages/native-stack/src"
+      ],
+      "@react-navigation/routers": [
+        "link:../react-navigation/packages/routers/src"
+      ],
+      "@react-navigation/stack": ["link:../react-navigation/packages/stack/"]
```

Comment:
Awesome :tada:

Reply:
Simplified it by a lot in the recent commit, check it out: https://github.com/software-mansion/react-native-screens/pull/2174/commits/9a6662e0895bc98a6509ac821f045c45e96f1e12